### PR TITLE
Adding info.

### PR DESCRIPTION
**Pull request by ChouaibB**
Hi,
In the following an additional clarification at the documentation of the function `plotAbundance` argument `order_rank_by` is added.
And, some changes at the examples comments.

### CODE REVIEW

**Review by tvborm — APPROVED** [review]

**Review by antagomir — COMMENTED** [review]
Ok - the grouping of taxa to top-5 + "Other" category not done yet in this PR?

**Review by FelixErnst — APPROVED** [review]

**Comment by ChouaibB** [comment]
> Ok - the grouping of taxa to top-5 + "Other" category not done yet in this PR?

You mean adding it to vignette?

**Comment by antagomir** [comment]
Top-N would make the example more clear. 

Suggestion:

Hmm ok keep "as is" in the vignette but add the example with "top-N" in OMA. 

Cite OMA in the vignette.

Show the OMA example also in the function help page (man/) because this is potentially common use case

We need to decide later, in general, how to split material between function manpages, vignettes, and OMA. This is a separate discussion that should be done elsewhere.

**Comment by ChouaibB** [comment]
> Top-N would make the example more clear.
> 
> Suggestion:
> 
> Hmm ok keep "as is" in the vignette but add the example with "top-N" in OMA.
> 
> Cite OMA in the vignette.
> 
> Show the OMA example also in the function help page (man/) because this is potentially common use case
> 
> We need to decide later, in general, how to split material between function manpages, vignettes, and OMA. This is a separate discussion that should be done elsewhere.

ok so I add the example of top-5 in vignette (since it's not there yet) and link to OMA where I make a top-N example?

**Comment by antagomir** [comment]
Sorry for being unclear. Top-N means top-5. Doesnt matter which number really, as long as the example is clear.

As follows:

- keep "as is" in the vignette (with no aggregation to "Other")
 
- add the example with "top-N" (top-5?) in OMA.

- Cite OMA in the vignette.

- Show the OMA example with top-N + "Other" also in the function help page (man/) because this is potentially common use case

We need to decide later, in general, how to split material between function manpages, vignettes, and OMA. This is a separate discussion that should be done elsewhere.

File: vignettes/miaViz.Rmd [hunk 1]
```diff
@@ -98,6 +98,8 @@ plots$abundance / plots$SampleType +
      plot_layout(heights = c(9, 1))
 ```
 
+Further example about composition barplot can be found at Orchestrating Microbiome Analysis [@OMA]; Microbiome Community.
```
Comment (antagomir) [inline vignettes/miaViz.Rmd:101]: Remove "Microbiome Community" (that, or relevant equivalent is already in the reference.

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
You can merge it after the last comment.

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**